sagemathinc/cocalc
# Fix missing test case

# Description
Adds the test case where uncommitted changes shows up and the connection is restored.

# Testing Steps
1. Open a course
1. Turn off your internet
1. Change something
1. Wait for "NOT Saved" to appear in the course
1. Turn on your internet connection.
1. After a little bit of time, the save button should go back to light green and no warning.

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):

Front end:
